### DISCUSSION TITLE
Show Popup over another popup

### DISCUSSION THREAD

**Discussion by gktval** [CommunityToolkit/Maui#703] [Q&A]
I have a popup displayed on a page. In certain circumstances I want to display another popup when the first is still being displayed. Ideally, the first popup would be invisible while the second popup is open. But I do not see a "Visible" Property in the popup. Is it possible to show a second popup when a first popup is already displayed?

**Comment by pictos** [comment]
AFAIK this is not supported on all platforms, I believe you can just do it on Android

**Comment by sjordanGSS** [comment]
Assuming the Maui Toolkit popup works in the same way as the Xamarin Toolkit popup, you can reset the dismissed-ness of a popup after dismissing it, allowing you to display and dismiss it again later. So you might be able to dismiss your first popup, ignore the result, then display it again once the second popup has been dismissed.
I used this a while ago to implement an interaction timeout that could would close an active popup and navigate to a countdown page, and then navigate back and display the same popup again if the countdown was cancelled. This sounds quite similar to what you're trying to achieve.

**Comment by gktval** [comment]
I tried something like that, but after dismissing the second popup, resetting the first popup, the first popup will not show again on the _page.ShowPupup(firstPopup);

**Comment by CliffAgius** [comment]
Closed and answered, but also has active discussion elsewhere.
